docs/admin/install/knative-with-operators.md

```diff
@@ -3,8 +3,8 @@
 Knative provides a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to install, configure and manage Knative.
 You can install the Serving component, Eventing component, or both on your cluster.
 
-!!! warning
-    The Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used for development or test purposes only.
+!!! important
+    The Knative Operator is now in Beta phase. Welcome to give it a test in your production environment.
```

Comment:
you could use the nice new feature state labels here, example at https://github.com/knative/docs/blob/4a12addcfe8c440ac456b8f72a76c55166685268/docs/developer/serving/services/custom-domains.md#configuring-custom-domains (will look like: https://knative.dev/docs/developer/serving/services/custom-domains/)